hera_cal/vis_clean.py

```diff
@@ -307,7 +372,12 @@ def vis_clean(self, keys=None, x=None, data=None, flags=None, wgts=None,
                   ax='freq', horizon=1.0, standoff=0.0, cache=None, mode='clean',
                   min_dly=10.0, max_frate=None, output_prefix='clean',
                   skip_wgt=0.1, verbose=False, tol=1e-9,
-                  overwrite=False, **filter_kwargs):
+                  overwrite=False,
+                  skip_flagged_edge_freqs=False,
```

Comment:
can you explain how these `skip_flagged_` params are different than the edgecut parameters?

Reply:
I'll explain this in the CLEAN context (iterative modeling with dft modes).

The edgecut parameters set the edge channels to zero which effects the CLEAN kernel, but does not affect the set of fourier modes that you are using to model your signal (using DFT modes spaced by 1/Ndata).  skip_flagged_edges only performs CLEAN on the regions bounded by entirely flagged edge channels which changes the basis that you are using to model your data (now you are modeling your data with DFT modes spaced by (1 / (Ndata - N_edge_skipped) ).  I've found that linear filters with DFT modes and DPSS modes tend to have problems with extrapolating flagged edges and also interpolating flags that are close to an edge when there is not taper that goes to zero. TBH I'm not completely sure why.